_posts/2018-12-09-real-world-salesforce-object-query-language-soql-examples.md

```diff
@@ -0,0 +1,45 @@
+Real-world Salesforce Object Query Language (SOQL) examples
+===========
+by Cody Goodman (@codygman)
+
+Have you ever spent more time than you wanted trying to figure out what you think should be a very simple problem? While [pairing](TODO) another developer and I experienced this problem, and thought it would be helpful to share the results of our problem solving to save others time.
```

Comment:
```suggestion
Have you ever spent more time than you wanted trying to figure out what you think should be a very simple problem? While [pairing](https://engineering.itpro.tv/2018/10/24/how-we-work-pair-programming/) another developer and I experienced this problem, and thought it would be helpful to share the results of our problem solving to save others time.
```

Reply:
Alternatively you could use the [promiscuous pairing link](https://engineering.itpro.tv/2018/12/07/our-first-experiment-with-promiscuous-pairing-is-a-success/) if you wanted.